lib/bootstrap.dart

```diff
@@ -146,6 +147,9 @@ Future<void> bootstrap(FutureOr<Widget> Function() builder) async {
               hiveRepository: getIt(),
             ),
           ),
+          BlocProvider<SearchCubit>(
+            create: (_) => getIt<SearchCubit>(),
+          ),
         ],
```

Comment:
This isn't properly registered. This wouldn't work well. Please refer to the exact way of how other cubits are registered.

Reply:
done